spec/e2e/enum_spec.rb

```diff
@@ -70,6 +70,16 @@
       expect(ids).to include(file2.id)
     end
 
+    it 'supports arrays' do
```

Comment:
Ah, I see now ;)  That spec right there next to the spec that I wrote! ;)

I removed my spec and used yours, though I added the third file again so that it could test that querying with an array will return multiple objects of different `type`s